dkist/dataset/tests/test_dataset.py

```diff
@@ -51,13 +52,15 @@ def test_repr(dataset, dataset_3d):
 
 
 def test_wcs_roundtrip(dataset):
+    # TOO generalise this so mark.accept_cli_dataset will work, if possible
     p = (10*u.pixel, 10*u.pixel)
     w = dataset.wcs.pixel_to_world(*p)
     p2 = dataset.wcs.world_to_pixel(w)
     assert_quantity_allclose(p, p2 * u.pix)
 
 
 def test_wcs_roundtrip_3d(dataset_3d):
+    # TOO generalise this so mark.accept_cli_dataset will work, if possible
```

Comment:
I don't know what this test is doing, but if we generalise the one above this one can stay here and be weird by itself.

```suggestion
```